toolkit-testing/shared/src/test/scala/org/typelevel/toolkit/ToolkitCompilationTest.scala

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2023 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.toolkit
+
+import munit.{CatsEffectSuite, TestOptions}
+import cats.effect.IO
+import buildinfo.BuildInfo.scalaVersion
+
+class ToolkitCompilationTest extends CatsEffectSuite {
+
+  testCompilation213("Toolkit should compile a simple Hello Cats Effect") {
```

Comment:
Is it worthwhile to test compile and running separately? If so, then maybe we should do it without so much duplication.